examples/stockticker/stockticker.slint

```diff
@@ -0,0 +1,46 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: MIT
+
+import { Button, VerticalBox, HorizontalBox } from "std-widgets.slint";
+
+export struct Symbol {
+    name: string,
+    price: float,
+}
+
+export component MainWindow inherits Window {
+    in-out property <[Symbol]> stocks: [
+        {
+            name: "AAPL",
+            price: 237.96,
+        },
+        {
+            name: "MSFT",
+            price: 509.24,
+        },
+        { name: "AMZN", price: 232.94 },
```

Comment:
I suppose this default value was used to develop the UI in the first place, but now it's kind of useless because it's immediately replaced with the contents of the model coming from the backend? How about making this an empty array here?

(as is, it's a bit confusing because "AMZN.US" != "AMZN" and because this data is - if I'm not missing anything - basically not used)